neonvm/tools/vm-builder-generic/main.go

```diff
@@ -20,13 +20,13 @@ import (
 	"github.com/docker/docker/client"
 )
 
-// vm-builder-generic --src alpine:3.16 --dst vm-alpine:dev --file vm-alpine.qcow2
+// vm-builder-generic --src alpine:3.18 --dst vm-alpine:dev --file vm-alpine.qcow2
 
 const (
 	dockerfileVmBuilder = `
 FROM {{.SrcImage}} AS rootdisk
 
-FROM alpine:3.16 AS vm-runtime
+FROM alpine:3.18 AS vm-runtime
 # add busybox
 ENV BUSYBOX_VERSION 1.35.0
```

Comment:
busybox v1.35.0 used in alpine:3.16 but in 3.18 we have busybox 1.36.1
may be update it too ?

Reply:
Ah, thanks, updated!

Reply:
Busybox doesn't provide binaries for 1.36.1 (https://busybox.net/downloads/binaries/), so I've switched it to installing `busybox` from Alpine.